submit-web/src/routes/proponent/_proponentLayout/projects/index.tsx

```diff
@@ -42,7 +42,7 @@ export function ProjectsPage() {
   return (
     <PageGrid>
       <Grid item xs={12}>
-        <ProjectFilters />
+        <ProjectFilters userType='proponent'/>
```

Comment:
This is a magic string issue, minor but better improved.
There's a constant for this, look for USER_TYPE.